src/client/common/application/notebook.ts

```diff
@@ -95,9 +95,9 @@ export class VSCodeNotebook implements IVSCodeNotebook {
         viewType: string,
         label: string,
         handler?: NotebookExecuteHandler,
-        preloads?: NotebookKernelPreload[]
+        rendererScripts?: NotebookRendererScript[]
```

Comment:
Trying to remember to update variable names as I do renames like this, just to keep the naming clearer when types change.